documentation/source/Introduction/pages/installation.rst

```diff
@@ -34,42 +34,34 @@ can be installed by appending ``[option]``, i.e. ``pip install NuRadioMC[option]
 
 Development version
 -------------------
-The most recent version of ``NuRadioMC`` is available on `github <https://github.com/nu-radio/NuRadioMC.git>`__.
-It can be downloaded manually from the `repository website <https://github.com/nu-radio/NuRadioMC/releases/latest>`__,
-or cloned using ``git``
+If you want the most recent, in-development version of ``NuRadioMC``, or intend to :doc:`contribute to its development </Introduction/pages/contributing>`,
+you can get it via `the NuRadioMC github <https://github.com/nu-radio/NuRadioMC.git>`__:
 
 .. code-block:: Bash
 
   git clone https://github.com/nu-radio/NuRadioMC.git
 
-If you don't already have it installed, you should `install Git <https://git-scm.com/>`_. 
-To set up NuRadioMC and install the required dependencies, navigate to the ``NuRadioMC`` folder and run the python installer provided:
+If you don't already have it installed, you should `install Git <https://git-scm.com/>`_.
 
-.. code-block:: bash
-
-  cd NuRadioMC/
-  python3 install_dev.py
+.. Note::
 
-This will launch an interactive installer that allows you to install all core and some optional dependencies, as well as setting up some git settings
-in case you want to contribute to NuRadioMC. **We highly recommend installing NuRadioMC inside a** `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_. 
-You can either use ``python3 -m venv name_of_venv``
-or use a virtual environment manager like `conda <https://anaconda.org/anaconda/python>`_. 
-If you for some reason do not want to use a virtual environment, you can install the dependencies for the current user only by appending 
-``python3 install_dev.py --user``.
+  **We highly recommend installing NuRadioMC inside a** `virtual environment <https://docs.python.org/3/tutorial/venv.html>`_.
+  You can either use ``python3 -m venv name_of_venv``
+  or use a virtual environment manager like `conda <https://anaconda.org/anaconda/python>`_.
 
-If the installer above does not work, or you want/need to install additional dependencies, 
-please follow the :ref:`manual installation instructions <Introduction/pages/installation:Manual installation>` below.
+To install NuRadioMC and its dependencies, use the `pip editable install <https://pip.pypa.io/en/stable/topics/local-project-installs/#editable-installs>`__.
+Navigate to the ``NuRadioMC`` folder and run:
 
-PYTHONPATH
-__________
-To use the development version of NuRadioMC, its installation path needs to be included in the user PYTHONPATH.
-If the installer does not succesfully do this, please manually add
-
-.. code-block:: Bash
+.. code-block:: bash
 
-  export PYTHONPATH=/path/to/NuRadioMC:$PYTHONPATH
+  cd NuRadioMC/
+  pip install -e .[dev]
+  pre-commit install
```

Comment:
can we configure the pyproject.toml such that `pre-commit install` is always executed?

Reply:
No, see comment above.